Doc/library/enum.rst

```diff
@@ -925,3 +925,17 @@ Notes
        >>> from enum import IntEnum
        >>> class MyIntEnum(IntEnum):
        ...     __str__ = IntEnum.__str__
+
+   Additionally, it's possible to end up shadowing a built-in function from
```

Comment:
`.title` is not a builtin function. It is a method of `str`.

Reply:
Gasp, I had it as method 😢 originally. Dunno why I changed it. Fixed in afde363e70c626697e04e141359413caff0d708e